pages/login.tsx

```diff
@@ -1,7 +1,7 @@
-import Head from "next/head"
-import Link from "next/link"
-import { FormButton, FormCard, FormInput } from "@components/FormComponents"
-import { Layout } from "@components/Layout"
+import Head from "next/head";
+import Link from "next/link";
```

Comment:
perhaps we have different linting/save preferences (i tend not to use semi colons)

Reply:
Yh we do. I added a `.prettierrc` config file so the project can have the same linting seetings